### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by DannyBen** [chjj/ttystudio#33]
Seems like issues are open for several years, and it fails to install.

if it is abandoned, perhaps mention it somewhere in the README and archive it?
it is still being recommended by some, and I remember that it worked well when it did, so I would love for it to *not* be abandoned...

**Comment by samtay** [comment]
Agreed, definitely think the author should take just a few minutes to mark as no longer supported, would have saved me quite some time tonight!

However, for those looking to *still* install this package (IMO it was the best terminal recorder available), a simple fix (at the time of this writing) is to use the `n` package, e.g.

```shell
npm install -g n
n install 8.17.0
n exec 8.17.0 npm install -g ttystudio
```

since the offending dependency pty.js builds OK with the now deprecated nodejs version 8.x.

**Comment by zicklag** [comment]
Thanks for the tip @samtay.

Here's a very nice alternative that worked great for me: https://github.com/nbedos/termtosvg.

**Comment by dionysius** [comment]
Can confirm that my issue was as well that this project requires an older node version. In my case, I'm using `nvm` to manage different versions of installed nodes:
```cmd
$ nvm install lts/carbon
Downloading and installing node v8.17.0...
...
$ nvm use lts/carbon
Now using node v8.17.0 (npm v6.13.4)
$ npm install -g ttystudio
...
added 6 packages from 10 contributors in 4.86s
```

**Comment by akashagarwal7** [comment]
Thanks heaps for the NVM native solution @dionysius!
